### DISCUSSION TITLE
System Tray and/or Toolbars on multiple monitors

### DISCUSSION THREAD

**Discussion by IZeeLikeEyeZee** [valinet/ExplorerPatcher#1549] [Ideas]
Idk if this is possible with how y'all change up the taskbar and stuff since ik its not possible natively, but it would be awesome if we could have the system tray and toolbars on multiple monitors, since often im gaming or whatever and it makes toolbars and the system tray pretty inaccessible. Changing primary monitors is not an option since then a bunch of games open on my second monitor and can be hard to get to my main monitor. Even if it was possible to just like duplicate the taskbar across all monitors would be great!

**Comment by NoReasonKen** [comment]
+1, it would be nice if there is an option to duplicate taskbar on all monitors (including notification panel)

**Comment by pyrates999** [comment]
This sounds like something Microsoft would do.

**Comment by shadedurza** [comment]
As someone with 4 monitors its strange to me that I have to choose only one to have the System Tray by default. I know one as to be the "main" taskbar but why can't they all look / function the same? I can have time on all of them (and clicking the time does the same thing on all of them) but not volume? This is exactly the kind of thing ExplorerPatcher is made to solve so big +1 from me.

**Comment by pyrates999** [comment]
Try this: https://gearupwindows.com/how-to-show-the-taskbar-on-all-monitor-on-windows-11/

Although it indicates that the system tray will only be shown on your primary monitor.  Without code existing that would show the system tray on all monitors, explorer patcher cannot enable such a feature.

**Reply by shadedurza** [reply]
Anyone can feel free to correct me here but I truly don't think those in this thread want just the taskbar on multiple monitors. We got that. It's a native thing. We're looking for the "System Tray" to be on multiple monitors. Things that aren't "the time" on more than one monitor. We aren't looking for this "feature to be enabled". We want this feature to be added. Via new code. If we could enable this feature we wouldn't be on the github for a 3rd party app. We would be in windows. Enabling the feature.

**Reply by pyrates999** [reply]
Explorer patcher doesn't add new features.  It enables existing features that Microsoft has decided to disable while still leaving the code intact.

Perhaps you can look at https://windhawk.net instead.

**Reply by shadedurza** [reply]
"ExplorerPatcher aims to enhance the working environment on Windows."
...
This request would enhance the working environment on Windows.
...
ExplorerPatcher adds plenty of new features. As long we're providing links nobody asked for you can review this if you like
https://github.com/valinet/ExplorerPatcher/wiki/All-features

**Reply by pyrates999** [reply]
All of these features exist because Microsoft disabled the features, but did not remove the code for them.

**Reply by shadedurza** [reply]
All those features exist because of the aim to enhance the working environment on Windows.
Also, which version of windows had the option for "opening the start **menu** on the monitor containing the cursor" again?
Which version of windows literally had the "Simple Window Switcher" which is quote "My own take on implementing this kind of functionality"  ?
Please ensure you provide exact build numbers so your replies can be verified.

**Reply by pyrates999** [reply]
You have access to the source code.  You try adding it.

**Reply by shadedurza** [reply]
I'd rather request it. Thanks for your attempt at help. I'm sure you did your best.

**Comment by Mapienator** [comment]
is this something that is still being worked on or should i give up at this point? :3
using a 3 monitor setup with middle primary is really annoying if you want to use system tray icons. Like the audio one is so important, yet so hard to access. (yes i have a disappearing taskbar :P )

<3<3<3
